src/c_api/c_api_utils.h

```diff
@@ -71,6 +70,7 @@ inline void CalcPredictShape(bool strict_shape, PredictionType type, size_t rows
     }
     break;
   }
+  case PredictionType::kApproxInteraction:
```

Comment:
Should something be happening here?

Reply:
No, the output shape is exactly the same between shap and approx.